### ISSUE TITLE
Module not found: Error: Can't resolve 'ngx-router-animations/public_api'

### ISSUE THREAD

**Issue by mahrmediait** [vugar005/ngx-router-animations#1]
Hi,

have anyone an idea how to handle this error?

Module not found: Error: Can't resolve 'ngx-router-animations/public_api'

Best greetings-

**Comment by vugar005** [comment]
Hi @mahrmediait 
You should import dependecies from `ngx-router-animations ` instead of `ngx-router-animations/public_api`
Please let me know if it solved issue.

**Comment by mahrmediait** [comment]
Hi @vugar005,

thanks for your answer! Not it told me that he can't find the module.

**Comment by vugar005** [comment]
@mahrmediait 
I know but that error is because you import animations incorrectly. 
You should replace as mentioned above.
Hope that helped.

**Comment by mahrmediait** [comment]
@vugar005 
I have already replaced it.

```
import { Component, OnInit } from '@angular/core';
import { rotateCubeToLeft } from 'ngx-router-animations';
import {transition, trigger, useAnimation} from '@angular/animations';

@Component({
  selector: 'app-website',
  templateUrl: './website.component.html',
  styleUrls: ['./website.component.scss'],
  animations:  [
    trigger('rotateCubeToLeft',  [ transition('home => about', useAnimation(rotateCubeToLeft))])
  ]
})
export class WebsiteComponent implements OnInit {

  constructor() { }

  ngOnInit() {
  }

  getState(outlet)  {
    return outlet.activatedRouteData.state;
  }
}
```

Compiler says:

ERROR in src/app/website/website.component.ts(2,34): error TS2307: Cannot find module 'ngx-router-animations'.

**Comment by vugar005** [comment]
@mahrmediait  could you please provide info for version of Angular, @angular/cli and ngx-router-animations version?

**Comment by vugar005** [comment]
@mahrmediait  Yes it is an issue in 0.0.8. Please install version 0.0.7 for now by running:
`npm install ngx-router-animations@0.0.7`

**Comment by mahrmediait** [comment]
@vugar005 

Have installed version 0.0.8 on angular 7.2.1

**Comment by vugar005** [comment]
Solved from version 0.0.9.  Thanks for informing. Closing issue
